modules/exploits/windows/ftp/open_ftpd_wbem.rb

```diff
@@ -134,12 +134,17 @@ def exploit
 
     begin
       t = framework.threads.spawn("reqs", false) {
-        # Upload our malicious executable
-        u = upload(exe_name)
-        # Upload the mof file
-        upload(mof_name) if u
-        register_file_for_cleanup("#{::File.basename(exe_name)}")
-        register_file_for_cleanup("wbem\\mof\\good\\#{::File.basename(mof_name)}")
+        begin
+          # Upload our malicious executable
+          u = upload(exe_name)
+          # Upload the mof file
+          upload(mof_name) if u
+          register_file_for_cleanup("#{::File.basename(exe_name)}")
+          register_file_for_cleanup("wbem\\mof\\good\\#{::File.basename(mof_name)}")
+        rescue ::Errno::ECONNRESET => e
```

Comment:
Eyes on here, should this be expanded to capture `::Exception`?
```suggestion
        rescue ::Exception => e
```

The `cleanup` call triggers the service from the mixin to exit `wait` state and cleanly exit the module.

Reply:
Further testing showed that `::Exception` is the better choice.